Automattic/wp-calypso
# My Email: Whitelist inbox paths for domain-only paths

#### Changes proposed in this Pull Request

* Add `/inbox` paths to the array of allowed paths

#### Testing instructions



* Go to https://wordpress.com/domains/ to create a domain-only site, or not if you already have one
* Add an email subscription to your domain-only site, with at least one mailbox
* Navigate to [Inbox](/inbox/<site-slug>) on your domain-only site
* Confirm that you are shown a domain-only empty content page with CTA buttons 
* <img width="400" alt="Screenshot 2021-09-30 at 2 52 45 AM" src="https://user-images.githubusercontent.com/277661/135372732-bc0b194c-b092-467d-b480-dd7bf0777250.png">
* Checkout this branch
* Navigate to [Inbox](/inbox/<site-slug>) on your domain-only site, again
* Confirm that this time around your inbox is shown with the mailbox(es) your purchased previously
